k8s.io/configmap-nginx.yaml

```diff
@@ -90,6 +90,19 @@ data:
         rewrite ^/(.*)?$    https://github.com/kubernetes/kubernetes/tree/master/$1 redirect;
       }
       server {
+        server_name dl.k8s.io dl.kubernetes.io;
+        listen 80;
+        listen 443 ssl;
+
+        location / {
+          # Don't require /release/ if you want to get at the Kubernetes release artifacts, the common case.
+          rewrite ^/(v[0-9]+\.[0-9]+.*)$   https://storage.googleapis.com/kubernetes-release/release/$1 redirect;
```

Comment:
yeah, this was me mostly being lazy. I wanted to not accidentally catch something like /v1/, though nothing in `gs://kubernetes-release/` matches anything like that right now, so just `/v[0-9]+` would work.

it looks like everything in `gs://kubernetes-release/release` is of the form `vX.X.X(-(alpha|beta).X)?`. if you want me to make the regex more specific, I can do that.